src/ext/background.js

```diff
@@ -90,6 +91,15 @@ let optEndpoints = () => activeConfig()["opt-endpoints"];
 let emptyRespHeaders = () => activeConfig()["spend-action"]["empty-resp-headers"];
 let storedCommitments = () => activeConfig()["commitments"];
 
+
+/**
+ * Allows access to get-more-passes-url from UpdatePopup
+ * @return {string}
+ */
+function getMorePassesUrl() {
```

Comment:
Nit: would it be possible to style this in the same as the other configuration functions above?

Reply:
I had issues with using it in the `popup` context, thats why its using the canonical form, I can spend some time figuring it out, its bugging me as well :)